src/components/selection_input/selection_input.ts

```diff
@@ -126,6 +131,54 @@ export class SelectionInput extends Component<Props, SpreadsheetChildEnv> {
     });
   }
 
+  startDragAndDrop(rangeId: number, event: MouseEvent) {
+    if (event.button !== 0 || (event.target as HTMLElement).tagName === "SELECT") {
+      return;
+    }
+
+    const rects = this.getDimensionElementsRects();
+    const draggableIds = this.ranges.map((range) => range.id);
+    const offset = 1; // column title
+    const draggableItems = draggableIds.map((id, index) => ({
```

Comment:
... is there a column title in the selection input ? :thinking:

Reply:
No ... This was the reason the behavior was buggy, nice catch. Thanks !